deploy/osd-delete-backplane-serviceaccounts/00-delete-backplane-serviceaccounts.rbac.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+  namespace: backplane
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: osd-delete-serviceaccounts
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - delete
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: osd-delete-backplane-serviceaccounts
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: osd-delete-serviceaccounts
+subjects:
+  - kind: ServiceAccount
+    name: osd-delete-backplane-serviceaccounts
+    namespace: backplane
```

Comment:
Should limit to backplane namespaces only.  Can use SubjectPremissions to manage this so it is based on regex.